plugins/@grouparoo/salesforce/src/lib/export/salesforceExporter.ts

```diff
@@ -338,6 +342,25 @@ function buildUserPayload(
   return { row, referenceData };
 }
 
+function getForeignKeyType(users: BatchExport[], recordMatchField: string) {
+  if (users.length > 0) {
```

Comment:
this could be `null` value. It's probably better to get it from the (cache, if not already) field definition itself: the same one used in `destinationMappings`.